docs/sdk-docs/connecting/README.md

```diff
@@ -1,23 +1,106 @@
-# Connecting to a Test Network
+# Connecting
+The SDK provides out of the box support for the deployed networks on `Görli`, `Ropsten`, `Rinkeby`, and `Mainnet`.
 
-Connecting to a Raiden test network is automatically done by the Light Client SDK. The SDK will automatically connect to the proper Raiden network based on the detected network id on your web3 provider.
+If you need to use the sdk on a private network, or a custom deployment you can find more information in the following [guide](../private-chain/README.md).
 
-The SDK provides out of the box support for the deployed networks on Ropsten, Rinkeby, and Mainnet.
+## Raiden Light Client test environment
+For development purposes, the Light Client uses a standalone environment. The dApp deployment [lightclient.raiden.network](https://lightclient.raiden.network/) and the development version served by 'npm run serve' also conforms to this configuration. 
 
-If you want you can also use the Light Client SDK to connect to networks that are not officially supported.
+This environment uses:
 
-To connect for example on a privately deployed Raiden network,
-you can initialize the SDK as usual, and pass the Contract Info as the fourth parameter of the create function.
+- A specific version of [Raiden](https://github.com/raiden-network/raiden/commit/2e741dfdf4bfa564dec760abd5e3d8b2c9d30715)
+- A matrix transport server - `https://raidentransport.test001.env.raiden.network`
+- A PFS server - `https://pfs.raidentransport.test001.env.raiden.network`
```

Comment:
Do you think they would provide any benefit? For me it would not make sense to navigate to the pfs and transport. I would just want to copy/paste. Having them as links might also enable some accidental click when you try to copy/paste them.

Reply:
Copying is way easier when they're links instead of code blocks :)

Reply:
You mean opening a new tab and then copying from the tab address bar?

Reply:
Or right click > copy link address :)
But not important it's fine as it is 👍